### PR TITLE
csh: add French translation

### PR DESCRIPTION
**Pull request by 879hun675tgfoO098**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/common/csh.md/csh.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+`csh -f`
+
+- Exécuter une commande:
+
+`csh -c "{{echo 'Exécution de la commande echo par csh'}}"`
+
+- Exécuter un script:
+
+`csh {{chemin/d_acces_au/script.csh}}`
```
Comment (kbdharun) [inline pages.fr/common/csh.md/csh.md:21]:
```suggestion
`csh {{chemin/vers/script.csh}}`
```

Referring the structure at 

https://github.com/tldr-pages/tldr/blob/99ecf782b809f72e55c434105c53dceb372f4eb7/contributing-guides/translation-templates/common-arguments.md?plain=1#L18

File: pages.fr/common/csh.md/csh.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# csh
+
+> Le shell (interprèteur de commandes) affiche une syntaxe proche de C.
+> Voir également : tcsh.
```
Comment (kbdharun) [inline pages.fr/common/csh.md/csh.md:4]:
```suggestion
> Voir aussi : tcsh.
```

[This](https://github.com/search?q=repo%3Atldr-pages%2Ftldr+Voir+aussi&type=code) seems to be used [more than "Voir également"](https://github.com/search?q=repo%3Atldr-pages%2Ftldr%20Voir%20%C3%A9galement&type=code) across our French pages.

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

**Comment by 879hun675tgfoO098** [comment]
Hello,

I’m sorry to bother you, but I don’t understand how to save my changes.

I’m in the right branch, I try to delete the file pages.fr/common/csh.md/sch.md and recreate it as pages.fr/common/csh.md with the modifications you suggested, but I get an error message that I do not understand: "There was an error committing your changes: File could not be edited”

What I have found on the internet regarding editing files in place on GitHub does not help.

Seems I have yet more studying to do before being able to contribute effectively.

Best regards,

YM

**Comment by kbdharun** [comment]
@879hun675tgfoO098 Fixed it for you.

File: pages.fr/common/csh.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# csh
+
+> Le shell (interprèteur de commandes) affiche une syntaxe proche de C.
+> Voir aussi : tcsh.
+> Plus d'informations : https://www.mkssoftware.com/docs/man1/csh.1.asp.
```
Comment (kbdharun) [inline pages.fr/common/csh.md:5]:
```suggestion
> Plus d'informations : <https://www.mkssoftware.com/docs/man1/csh.1.asp>.
```
We wrap our More information links inside `<>` (angular brackets). Once this suggestion is resolved the PR is GTG.

**Review by acuteenvy — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Comment by 879hun675tgfoO098** [comment]
Awesome, thank you. So, is this a success ? Am I officially a Github contributor ?

**Comment by kbdharun** [comment]
> Awesome, thank you. So, is this a success ? Am I officially a Github contributor ?

You indeed are when the PR gets merged after 1 more approval.

**Comment by kbdharun** [comment]
> Awesome, thank you. So, is this a success ? Am I officially a Github contributor ?

Yes, Congratulations on your first contribution.

**Pull request merged**